language-design.md

```diff
@@ -17,7 +12,49 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> "After all, the very existence
+of a master plan means, by definition, that the members of the community
+can have little impact on the future shape of their community, because most
+of the important decisions have already been made. In a sense, under a master
+plan people are living with a frozen future, able to affect only relatively 
+trivial details." [Steele, 1998]
+
+This quote by Christopher Alexander is something I definitely agree with. Before
+reading this article, I did not think about this much and I would have maybe 
+held the opinion that it would be good to have a master plan and have the team
```

Comment:
I think I still hold this opinion - but you have to be willing to adapt your master plan. In my mind, a master plan should be at most a roadmap with the currently envisioned end goall, but open to a new destination. Without some type of roadmap or goal it's easy to drift away. I would say this is especially true in a DSL, where there is a desired domain to reach.